docs/src/content/docs/guides/customization.mdx

```diff
@@ -406,7 +409,7 @@ It provides npm modules you can install for the fonts you want to use and includ
 To apply the font you set up to your site, use your chosen font’s name in a [custom CSS file](/guides/css-and-tailwind/#custom-css-styles).
 For example, to override Starlight’s default font everywhere, set the `--sl-font` custom property:
 
-```css
+```css {4}
```

Comment:
This one feels like it might be overkill given it’s almost highlighting the full example and the `:root` selector is relevant too: 

<img width="436" alt="three-line code block with one line highlighted" src="https://github.com/withastro/starlight/assets/357379/fe55288a-cb16-4cd5-9e52-c6b027290cfb">

Might suggest removing the highlight in this case:

````suggestion
```css
````